inference-engine/src/inference_engine/CMakeLists.txt

```diff
@@ -216,7 +217,7 @@ set_target_properties(${TARGET_NAME} ${TARGET_NAME}_obj ${TARGET_NAME}_s
 
 # Export for build tree
 
-export(TARGETS ${TARGET_NAME} NAMESPACE IE::
+export(TARGETS ${NGRAPH_LIBRARIES} ${TARGET_NAME} NAMESPACE IE::
```

Comment:
do we need it? Currently `InferenceEngineConfig.cmake` already includes `find_package(ngraph REQUIRED)`

Reply:
Looks like we don't need.

Reply:
Reverted, because some targets cannot find IE::ngraph.